WordPress/src/main/java/org/wordpress/android/ui/people/utils/PeopleUtils.java

```diff
@@ -115,6 +115,47 @@ public void onErrorResponse(VolleyError volleyError) {
         WordPress.getRestClientUtilsV1_1().get(path, params, null, listener, errorListener);
     }
 
+    public static void fetchViewers(final String blogId, final int localTableBlogId, final int offset,
+                                     final FetchViewersCallback callback) {
+        com.wordpress.rest.RestRequest.Listener listener = new RestRequest.Listener() {
+            @Override
+            public void onResponse(JSONObject jsonObject) {
+                if (jsonObject != null && callback != null) {
+                    try {
+                        JSONArray jsonArray = jsonObject.getJSONArray("viewers");
+                        List<Person> people = peopleListFromJSON(jsonArray, localTableBlogId, false, false,
+                                true);
+                        int numberOfUsers = jsonObject.optInt("found");
+                        boolean isEndOfList = (people.size() + offset) >= numberOfUsers;
+                        callback.onSuccess(people, isEndOfList);
+                    }
+                    catch (JSONException e) {
+                        AppLog.e(T.API, "JSON exception occurred while parsing the response for " +
+                                "sites/%s/viewers: " + e);
+                        callback.onError();
+                    }
+                }
+            }
+        };
+
+        RestRequest.ErrorListener errorListener = new RestRequest.ErrorListener() {
+            @Override
+            public void onErrorResponse(VolleyError volleyError) {
+                AppLog.e(T.API, volleyError);
+                if (callback != null) {
+                    callback.onError();
+                }
+            }
+        };
+
+        int page = (int) Math.floor(offset / FETCH_LIMIT) + 1;
```

Comment:
This being a division between integers, I'm not sure what the `Math.floor` usage is offering since the division will already "floor" the result and return an `int` anyway. What am I missing?